src/translator/service.h

```diff
@@ -38,18 +63,21 @@ class Service {
 
 public:
   /// @param options Marian options object
-  /// @param model_memory byte array (aligned to 64!!!) that contains the bytes
+  /// @param modelMemory byte array (aligned to 64!!!) that contains the bytes
   /// of a model.bin. Optional, defaults to nullptr when not used
-  explicit Service(Ptr<Options> options, const void *model_memory = nullptr);
+  /// @param shortlistMemory byte array of shortlist
+  explicit Service(Ptr<Options> options, const AlignedMemory* modelMemory, const AlignedMemory* shortlistMemory);
```

Comment:
It's really looking like we will own this memory, so I'd move them in here as members that might happen to be empty / null pointers.